tests/pytype_test.py

```diff
@@ -94,21 +96,19 @@ def run_pytype(*, filename: str, python_version: str, missing_modules: Iterable[
     return stderr
 
 
-def _get_relative(filename: str) -> str:
-    top = 0
+def _get_relative(filename: StrPath) -> Path:
+    filepath = Path(filename)
     for d in TYPESHED_SUBDIRS:
         try:
-            top = filename.index(d + os.path.sep)
+            return filepath.relative_to(Path(d).absolute().parent)
         except ValueError:
             continue
-        else:
-            break
-    return filename[top:]
+    raise ValueError(f"{filepath} not relative to {TYPESHED_SUBDIRS}")
```

Comment:
Could we just do instead:
```py
def _get_relative(filename: StrPath) -> Path:
    return Path(filename).relative_to(Path().absolute())
```

Reply:
Which makes `_get_relative` viable to be renamed `relative_to_typeshed_base` and moved to `ts_utils/path.py`